BEIMA.Backend.Test/MongoConnectorTest.cs

```diff
@@ -0,0 +1,206 @@
+﻿using NUnit.Framework;
+using BEIMA.Backend.MongoService;
+using System.IO;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System;
+using MongoDB.Bson;
+
+namespace BEIMA.Backend.Test
+{
+    [TestFixture]
+    public class MongoConnectorTest
+    {
+        //private MongoConnector? mongo;
+        class LocalSettings
+        {
+            public bool IsEncrypted { get; set; }
+            public Dictionary<string, string> Values { get; set; }
```

Comment:
This being null is making the github thing upset.

Reply:
Fixed be69b94